routing/road_graph_router.cpp

```diff
@@ -229,8 +229,8 @@ IRouter::ResultCode RoadGraphRouter::CalculateRoute(m2::PointD const & startPoin
     ASSERT_GREATER(result.distance, 0., ());
 
     CHECK(m_directionsEngine, ());
-    ReconstructRoute(*m_directionsEngine, *m_roadGraph, nullptr /* trafficColoring */,
-                     delegate, result.path, route);
+    ReconstructRoute(*m_directionsEngine, *m_roadGraph, nullptr, delegate, true /* hasAltitude */,
```

Comment:
So, previous version of this code passed false instead of true, and tests were happy, right?. Can we test this behavior?